setup.py

```diff
@@ -41,6 +41,7 @@
 setup_requires = []
 
 install_requires = [
+    'amqp===2.5.1',  # FIXME: required by kombu, Travis fails without it
```

Comment:
It's not the correct way to fix this. `amqp` 2.5.2 is forced by `kombu` version 4.6.5 which actually required `amqp` in version 2.5.1, its better to skip `kombu` in version 4.6.5 as the only change it introduces is `amqp` requirement updated from 2.5.1 to 2.5.2 https://github.com/celery/kombu/commit/b51d1d678e198a80d7e5fd95f32674c7d8e04a75#diff-f7b7288f81db184bb127adf1389dafc4